obsei/analyzer/ner_analyzer.py

```diff
@@ -100,3 +101,51 @@ def analyze_input(
                     )
                 )
         return analyzer_output
+
+
+class SpacyNERAnalyzer(BaseAnalyzer):
+    _nlp: en.English = PrivateAttr()
+    _max_length: int = PrivateAttr()
```

Comment:
How Spacy pipeline handle max_length case?

Reply:
Spacy pipeline by default has a large character limit, 1,000,000 characters . And that can be raised , long texts are handled internally. We can put a max_length limit just like in ClassificationAnalyzer (processing first max_length characters) if required.

Reply:
I am not aware about this. In this case let's remove `_max_length` variable at all